maps/bearcat/bearcat_define.dm

```diff
@@ -20,7 +20,7 @@
 	allowed_spawns = list("Cryogenic Storage")
 	default_spawn = "Cryogenic Storage"
 	use_overmap = 1
-	num_exoplanets = 2
+	num_exoplanets = 0
```

Comment:
Debugging code left in?

Reply:
yeah, gonna remove in the end, but it does speed up the init a lot